app/models/manageiq/providers/hawkular/middleware_manager/middleware_diagnostic_report.rb

```diff
@@ -88,14 +88,16 @@ def enqueue_job
 
       EmsEvent.add_queue(
         'add', ems_id,
-        :ems_id          => ems_id,
-        :source          => 'EVM',
-        :timestamp       => Time.zone.now,
-        :event_type      => 'hawkular_event',
-        :message         => _('Generation of JDR report was requested by a user.'),
-        :middleware_ref  => middleware_server.ems_ref,
-        :middleware_type => 'MiddlewareServer',
-        :username        => requesting_user
+        :ems_id                 => ems_id,
+        :source                 => 'EVM',
+        :timestamp              => Time.zone.now,
+        :event_type             => 'hawkular_event',
+        :message                => _('Generation of JDR report was requested by a user.'),
+        :middleware_server_id   => middleware_server.id,
+        :middleware_server_name => middleware_server.name,
```

Comment:
is this the only really changed line of code? the rest seems like formatting changes. if so, it's confusing, please leave only the "real" changed code